frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -89,20 +93,30 @@ const ManageSchedulePage = (): JSX.Element => {
     setShowRemoveScheduledQueryModal,
   ] = useState(false);
   const [selectedQueryIds, setSelectedQueryIds] = useState([]);
+  const [
+    selectedScheduledQuery,
+    setSelectedScheduledQuery,
+  ] = useState<IGlobalScheduledQuery>();
 
   const toggleScheduleEditorModal = useCallback(() => {
+    setSelectedScheduledQuery(undefined); // create modal renders
     setShowScheduleEditorModal(!showScheduleEditorModal);
   }, [showScheduleEditorModal, setShowScheduleEditorModal]);
 
   const toggleRemoveScheduledQueryModal = useCallback(() => {
     setShowRemoveScheduledQueryModal(!showRemoveScheduledQueryModal);
   }, [showRemoveScheduledQueryModal, setShowRemoveScheduledQueryModal]);
 
-  const onRemoveScheduledQueryClick = (selectedTableQueryIds: any) => {
+  const onRemoveScheduledQueryClick = (selectedTableQueryIds: any): any => {
     toggleRemoveScheduledQueryModal();
     setSelectedQueryIds(selectedTableQueryIds);
   };
 
+  const onEditScheduledQueryClick = (selectedQuery: any): any => {
```

Comment:
Just a note: nothing is being returned so `any` isn't necessary. If you want to specify, `void` is better.